build/print.go

```diff
@@ -514,12 +521,12 @@ func (p *printer) expr(v Expr, outerPrec int) {
 
 	case *BinaryExpr:
 		// Precedence: use the precedence of the operator.
-		// Since all binary expressions format left-to-right,
+		// Since all binary expressions FormatWithoutRewriting left-to-right,
```

Comment:
revert?

Please double-check all the other comments below, there seems to be some unintentional renaming.

Reply:
Thanks! That wasn't even Replace All, I used the Idea's refactoring tool to rename a function together with all its calls, but it also edits some random comments.

Reply:
Done.